Update Featured Artists and add Mappers' Guild (#3280)
ppy/osu-wiki: wiki/Featured_Artists/en.md

## Selection

Recent additions to the featured artist library are selected indirectly by the osu! community. If beatmappers create and rank maps for an artist, the osu! team may reach out to the artist to discuss the featured artist program. However, some artists are occasionally invited to become featured artists based on other factors, such as their expected appeal to osu!'s beatmapping audience.

Most songs in the featured artist program are non-exclusively licensed for osu!. In rare cases an artist will create tracks exclusively for osu!, which are labeled with "osu! exclusive" on their featured artist listing.

Details for a musician joining the featured artists can be found on the [music licensing](/wiki/Legal/Music_Licensing) article.

## Releases

New featured artist releases per year:

- **2016:** 8
- **2017:** 11
- **2018:** 20
- **2019:** 31
- **2020:** TBD

New featured artists are announced weekly on osu!'s news feed. Most announcements include a list of the artist's licensed songs and a link to their featured artist page, though some announcements are paired with mapping contests or beatmaps created by the [Mappers' Guild](/wiki/Mappers_Guild).

## Community usage

> Beatmappers are encouraged to create maps for featured artist tracks. These tracks are provided with timing data and have no risk of DMCA takedowns, allowing a user to immediately jump into mapping without worry. The work-in-progress development build of osu!, dubbed [osu!lazer](https://github.com/ppy/osu/releases), will mark these maps prominently in-game.

Reviewer: ```suggestion
Beatmappers are encouraged to create maps for featured artist tracks. These tracks are provided with timing data and are already cleared for use within osu!, allowing a user to immediately jump into mapping without worry. The work-in-progress development build of osu!, dubbed [osu!lazer](https://github.com/ppy/osu/releases), will mark these maps prominently in-game.
```
